javaslang/src/main/java/javaslang/collection/LinkedHashMap.java

```diff
@@ -149,6 +124,123 @@ private LinkedHashMap(Queue<Tuple2<K, V>> list, HashMap<K, V> map) {
     }
 
     /**
+     * Creates a LinkedHashMap of the given list of key-value pairs.
+     *
+     * @param <K> The key type
+     * @param <V> The value type
+     * @return A new Map containing the given entries
+     */
+    public static <K, V> LinkedHashMap<K, V> of(K k1, V v1, K k2, V v2) {
+        final HashMap<K, V> map = HashMap.of(k1, v1, k2, v2);
+        final Queue<Tuple2<K, V>> list = Queue.of(Tuple.of(k1, v1), Tuple.of(k2, v2));
+        return new LinkedHashMap<>(list, map);
```

Comment:
minor:
* consider declaring the variables in their order of usage
* it's weird to call a `Queue` `list` - though that's what the `LinkedHashMap` calls it also
* Consider using the `API` shortcuts for `HashMap.of`, `Queue.of` and `Tuple.of`, i.e. 
```java
Queue(Tuple(k1, v1), Tuple(k2, v2));
```